src/Umbraco.Web.UI.Client/src/packages/core/workspace/conditions/workspace-content-type-unique/workspace-content-type-unique.condition.ts

```diff
@@ -0,0 +1,54 @@
+import { UMB_CONTENT_WORKSPACE_CONTEXT } from '@umbraco-cms/backoffice/content';
+import type { UmbWorkspaceContentTypeUniqueConditionConfig } from './types.js';
+import type { UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
+import type { UmbConditionControllerArguments, UmbExtensionCondition } from '@umbraco-cms/backoffice/extension-api';
+import { UmbConditionBase } from '@umbraco-cms/backoffice/extension-registry';
+
+const ObserveSymbol = Symbol();
+
+/**
+ * Condition to apply workspace extension based on a content type unique (GUID)
+ */
+export class UmbWorkspaceContentTypeUniqueCondition
+	extends UmbConditionBase<UmbWorkspaceContentTypeUniqueConditionConfig>
+	implements UmbExtensionCondition
+{
+	constructor(
+		host: UmbControllerHost,
+		args: UmbConditionControllerArguments<UmbWorkspaceContentTypeUniqueConditionConfig>,
+	) {
+		super(host, args);
+
+		let permissionCheck: ((contentTypeUniques: string[]) => boolean) | undefined = undefined;
+		if (this.config.match) {
+			permissionCheck = (contentTypeUniques: string[]) => contentTypeUniques.includes(this.config.match!);
+		} else if (this.config.oneOf) {
+			permissionCheck = (contentTypeUniques: string[]) =>
+				contentTypeUniques.some((item) => this.config.oneOf!.includes(item));
+		}
+
+		if (permissionCheck !== undefined) {
+			this.consumeContext(UMB_CONTENT_WORKSPACE_CONTEXT, (context) => {
```

Comment:
I have compared this condition with the `ContentTypeAlias` Condition, and I can see that we use the `UMB_PROPERTY_STRUCTURE_WORKSPACE_CONTEXT` in that one. I'm not sure what the difference is between the two, other than `UMB_PROPERTY_STRUCTURE_WORKSPACE_CONTEXT` sounding more generic. Is there a reason you chose this one?

Reply:
I just double-checked with Niels. We can use the `UMB_PROPERTY_STRUCTURE_WORKSPACE_CONTEXT`. Then both the `alias` and `unique` conditions use the same token and behave the same.